python/src/opendp/polars.py

```diff
@@ -423,6 +423,57 @@ def release(self) -> OnceFrame:
             query = object.__getattribute__(self, "_query")
             resolve = object.__getattribute__(self, "resolve")
             return query._context(resolve())  # type: ignore[misc]
+        
+        def accuracy(self, alpha: float | None = None):
+            """Retrieve noise scale parameters and accuracy estimates for each output.
+
+            If ``alpha`` is passed, the resulting data frame includes an ``accuracy`` column.
+            
+            :example:
+
+            >>> import polars as pl
+            >>> data = pl.LazyFrame([pl.Series("convicted", [0, 1, 1, 0, 1] * 50, dtype=pl.Int32)])
+
+            >>> context = dp.Context.compositor(
+            ...     data=data,
+            ...     privacy_unit=dp.unit_of(contributions=1),
+            ...     privacy_loss=dp.loss_of(epsilon=1.0),
+            ...     split_evenly_over=1,
+            ...     margins={(): dp.Margin(max_partition_length=1000)},
+            ... )
+
+            >>> query = context.query().select(
+            ...     pl.len().dp.noise(), 
+            ...     pl.col("convicted").fill_null(0).dp.sum((0, 1))
+            ... )
+
+            >>> query.accuracy(alpha=.05)  # type: ignore[union-attr]
+            shape: (2, 5)
+            ┌───────────┬───────────┬─────────────────┬───────┬──────────┐
+            │ column    ┆ aggregate ┆ distribution    ┆ scale ┆ accuracy │
+            │ ---       ┆ ---       ┆ ---             ┆ ---   ┆ ---      │
+            │ str       ┆ str       ┆ str             ┆ f64   ┆ f64      │
+            ╞═══════════╪═══════════╪═════════════════╪═══════╪══════════╡
+            │ len       ┆ Len       ┆ Integer Laplace ┆ 2.0   ┆ 6.429605 │
+            │ convicted ┆ Sum       ┆ Integer Laplace ┆ 2.0   ┆ 6.429605 │
+            └───────────┴───────────┴─────────────────┴───────┴──────────┘
+
+            The accuracy in any given row can be interpreted with:
+
+            >>> def interpret_accuracy(distribution, scale, accuracy, alpha):
+            ...     return (
+            ...         f"When the {distribution} scale is {scale}, "
+            ...         f"the DP estimate differs from the true value by no more than {accuracy} "
+            ...         f"at a statistical significance level alpha of {alpha}, "
+            ...         f"or with (1 - {alpha})100% = {(1 - alpha) * 100}% confidence."
+            ...     )
+            ... 
+            >>> interpret_accuracy("Integer Laplace", 2.0, 6.429605, alpha=.05) # doctest:+SKIP
+            
```

Comment:
```suggestion
            >>> interpret_accuracy("Integer Laplace", 2.0, 6.429605, alpha=.05)
            ... 
            
```
Just so we don't get in the habit of skipping tests.